orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -1118,6 +1118,46 @@ void RegisterTrainingOpSchemas() {
           {"float"},
           "Constrain learning rate to float");
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(SGDOptimizerV2)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .Input(0, "lr", "The learning rate.", "T1")
+      .Input(1, "weights", "Sequence of weights to optimize.", "S_WEIGHT")
+      .Input(2, "gradients", "Sequence of gradients computed in this iteration.", "S_GRAD")
+      .Input(3, "update_signal",
```

Comment:
nit: rename to skip_update to make it more intuitive?

Reply:
oh, maybe the comment is confusing, update_signal means not to skip, so we cannot use skip_update. I have update the comment to make it clear.